src/assets/scss/assets/pages/_mainPage.scss

```diff
@@ -1,19 +1,74 @@
 @import '../variables/colors';
 
+@import "../components/header";
+@import "../components/menu";
+
 .mainPageBody {
     height: 100vh;
-    background-color: $body_color;
-
-    header {
-        background-color: $header_color;
-        height: 54px;
-        text-align: end;
+    background-color: white;
+    .status{
+        text-align: center;
+        margin-top: 8%;
     }
+}
+
+.role-container{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
+    margin-top: 3%;
+}
+
+.role-block{
```

Comment:
Role names, such as <Recipient>, are difficult to see

![Screenshot 2022-05-04 213727](https://user-images.githubusercontent.com/56841893/166803434-5bd4bdce-ec7d-40e4-9303-7aeb8db99687.jpg)